openapiart/common.py

```diff
@@ -398,8 +419,11 @@ def _set_property(self, name, value, choice=None):
             self._set_choice(name)
             self._properties[name] = self._DEFAULTS[name]
         else:
-            self._set_choice(name)
-            self._properties[name] = value
+            if not self._is_enum_valid(name, value):
```

Comment:
Not sure what is going wrong. But I am not getting any error whey try this
```python
pc = api.prefix_config()
pc.deserialize(payload)
pc.response = "abc"
pc.validate(True)    # return black though I am expecting enum error
```

Reply:
pc.validate(True) will skip raising the exception